server/webapp/WEB-INF/rails.new/app/views/layouts/single_page_app.html.erb

```diff
@@ -13,7 +13,7 @@
   <%= csrf_meta_tags %>
 </head>
 
-<body data-controller-name="<%= controller_name %>" data-action-name="<%= action_name %>">
+<body data-controller-name="<%= controller_name %>" data-action-name="<%= action_name %>" id="<%= controller_name %>-page">
```

Comment:
Can you please refer to https://github.com/gocd/gocd/pull/2868/files/6b7197e5af00cbf9c86b6bd780512218165ea463#diff-ece1b629796718ab9f569c1d688c0a49R16.